src/bin/deploy.rs

```diff
@@ -0,0 +1,34 @@
+use std::path::PathBuf;
+
+use lazy_static::lazy_static;
+use starknet_rs::{
+    services::api::contract_class::ContractClass, testing::starknet_state::StarknetState,
+    utils::Address,
+};
+
+#[cfg(feature = "with_mimalloc")]
+use mimalloc::MiMalloc;
+
+#[cfg(feature = "with_mimalloc")]
+#[global_allocator]
+static ALLOC: MiMalloc = MiMalloc;
+
+lazy_static! {
+    // include_str! doesn't seem to work in CI
```

Comment:
It turns out it _does_ work. It just wasn't working when I wrote the original comment, because contracts weren't being compiled. Now we should be able to use `include_str!` anywhere.